docs/architecture.md

```diff
@@ -1,188 +1,260 @@
-relevant Admission Review requests to the right `policy-server` endpoint.
+<!--TODO:I made endpoints plural. I think there are many. Is that correct?-->
+Now that all the necessary plumbing is complete,
+Kubernetes starts sending Admission Review requests to the right `policy-server` endpoints.
 
 ![Policy in action](/img/architecture_sequence_05.png)
 
-`policy-server` receives the Admission Request object and, based on the
-endpoint that received the request, uses the right policy to evaluate it.
+A `policy-server` receives the Admission Request object and,
+based on the endpoint that received the request,
+uses the correct policy to evaluate it.
 
-Each policy is evaluated inside of its own dedicated WebAssembly sandbox.
-The communication between `policy-server` (the "host") and the WebAssembly
-policy (the "guest") is done using the waPC communication protocol. This is
-covered in depth inside of [this](/writing-policies/index.md)
-section of the documentation.
+Each policy evaluates inside its own dedicated WebAssembly sandbox.
+The communication between `policy-server` (the "host")
+and the WebAssembly policy (the "guest")
+uses the waPC communication protocol.
+There is a description of using the protocol in the [writing policies](/writing-policies/index.md) documentation.
 
-## How multiple policy servers and policies are handled
+## How Kubewarden handles many policy servers and policies
 
-A cluster can have multiple policy servers and Kubewarden policies defined. 
+A cluster can have many policy servers and Kubewarden policies defined.
+There are benefits of having many policy servers:
 
-Benefits of having multiple policy servers:
-- Noisy Namespaces/Tenants generating lots of policy evaluations can be isolated from the rest of the cluster and do not affect other users.
-- Mission critical policies can be run inside of a Policy Server "pool", making your whole infrastructure more resilient.
+- You can isolate noisy namespaces or tenants,
+those generating many policy evaluations,
+from the rest of the cluster so as not to adversely affect other cluster operations.
 
-Each `policy-server` is defined via its own `PolicyServer` resource and each policy is defined via its own
-`ClusterAdmissionPolicy` resource. 
+- You can run mission-critical policies in a dedicated Policy Server pool,
+making your infrastructure more resilient.
+
+A `PolicyServer` resource defines each `policy-server`
+and a `ClusterAdmissionPolicy` or `AdmissionPolicy` resource defines each policy.
 
 This leads back to the initial diagram:
 
 ![Full architecture](/img/architecture.png)
 
-A `ClusterAdmissionPolicy` is bounded to a `PolicyServer`. `ClusterAdmissionPolicies` that don't specify any `PolicyServer`
-will be bounded to the `PolicyServer` named `default`. If a `ClusterAdmissionPolicy` references a `PolicyServer` that doesn't
-exist, it will be in an `unschedulable` state.
+<!--TODO:is/are bound or binds? What does the binding and how?-->
```

Comment:
The same of what I've already explain in a previous comment. I'll skip this one just to make things easier. Let me know if you think that explanation is not good enough.

Reply:
That's fine